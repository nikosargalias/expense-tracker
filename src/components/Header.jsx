import React from "react";
import { Link, NavLink } from "react-router-dom";

const Header = () => (
  <header>
    <ul>
      <li>
        <NavLink activeClassName="is-active" exact={true} to="/">
          Home
        </NavLink>
      </li>
      <li>
        <NavLink activeClassName="is-active" exact={true} to="/AddExpense">
          Add Expense
        </NavLink>
      </li>
      <li>
        <NavLink activeClassName="is-active" exact={true} to="/Help">
          Help
        </NavLink>
      </li>
      <li>
        <NavLink activeClassName="is-active" exact={true} to="/dashboard">
          Dashboard
        </NavLink>
      </li>
    </ul>
  </header>
);

export default Header;
