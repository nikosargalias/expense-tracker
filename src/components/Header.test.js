import React from "react";
import { shallow } from "enzyme";
import Header from "./Header";

test("should render Header correctly", () => {
  const wrapper = new shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
