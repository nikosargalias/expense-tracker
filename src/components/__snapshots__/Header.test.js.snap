// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <ul>
    <li>
      <NavLink
        activeClassName="is-active"
        exact={true}
        to="/"
      >
        Home
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        exact={true}
        to="/AddExpense"
      >
        Add Expense
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        exact={true}
        to="/Help"
      >
        Help
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        exact={true}
        to="/dashboard"
      >
        Dashboard
      </NavLink>
    </li>
  </ul>
</header>
`;
